zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -49,6 +50,33 @@ def __init__(self, read_key, scalyr_region=None):
     def count(self, query, minutes=5):
         return self.timeseries(query, function='count', minutes=minutes, buckets=1, prio='low')
 
+    def messages(self, query, max_count=100, minutes=5, continuation_token=None):
+
+        val = {
+            'token': self.__read_key,
+            'queryType': 'log',
+            'maxCount': max_count,
+            'filter': query,
```

Comment:
I am thinking that validating ``query`` **not** to be empty string would help in narrowing down the queries and prevents from making mistakes of retrieving/processing all account logs.

Reply:
Validation has been added tested